pkg/devfile/parser/parse.go

```diff
@@ -217,6 +165,11 @@ func ParseDevfile(args ParserArgs) (d DevfileObj, err error) {
 		d.Ctx.SetToken(args.Token)
 	}
 
+	args.DevfileUtilsClient = NewDevfileUtilsClient()
```

Comment:
so if we are setting it here does this mean, when HAS test calls `ParseDevfile()` it will not use the mock impl?

Reply:
Good catch, I will need to initialize it only when it's not initialized